.ci/magic-modules/generate-terraform-beta.yml

```diff
@@ -0,0 +1,24 @@
+---
+# This file takes two inputs: magic-modules-branched in detached-HEAD state, and the list of patches.
+# It spits out "terraform-generated", a terraform repo on a new branch (named after the
```

Comment:
Comment needs updating.  (I wish this file wasn't necessary, ugh, seems like a big pain for no good reason, but it is since the output name changes)